### PR TITLE
init label get cmd

### PR DESCRIPTION
**Pull request by ransomw1c**
implement `label get` per #231

### CODE REVIEW

**Comment by ransomw1c** [comment]
current demo

```shell
cmd/datamon% go build && ./datamon label get --repo ransom-datamon-test-repo --label asdf ; echo "$?" && ./datamon label get --repo ransom-datamon-test-repo --label testlabel ; echo "$?"
```

**Comment by ransomw1c** [comment]
leaving the WIP tag on this to hopefully rebase and dedupe after #224 ...

... also, want to pause for particular opinions about what to print in the case of an extant label.  same format as `label list`?

**Comment by kerneltime** [comment]
> #224



> leaving the WIP tag on this to hopefully rebase and dedupe after #224 ...
> 
> ... also, want to pause for particular opinions about what to print in the case of an extant label. same format as `label list`?
Sounds good.

File: cmd/datamon/cmd/label_get.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+)
+
+var GetLabelCommand = &cobra.Command{
+	Use:   "get",
+	Short: "Get bundle info by label",
+	Long: `Performs a direct lookup of labels by name.
+Prints corresponding bundle information if the label exists,
+exits with ENOENT status otherwise.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if params.repo.ContributorEmail == "" {
```
Comment (kerneltime) [inline cmd/datamon/cmd/label_get.go]: Thought not enforced uniformly, we should allow read operations without contributor information being set.

Comment (ransomw1c) [reply cmd/datamon/cmd/label_get.go]: yes, this was removed altogether

File: pkg/storage/gcs/store.go [hunk 2]
```diff
@@ -58,7 +58,7 @@ func (g *gcs) Has(ctx context.Context, objectName string) (bool, error) {
 	}
```
Comment (kerneltime) [inline pkg/storage/gcs/store.go]: The method should be updated to use the client that is part of the [gcs instance](https://github.com/oneconcern/datamon/pull/233/files#diff-aaf1bc657e525c33e0d1422846c07b12R97)

**Pull request merged**